Makefile

```diff
@@ -1,6 +1,7 @@
 # Define the target to run if make is called with no arguments.
 default: check
 
+LOG_LEVEL?=fatal
```

Comment:
`export LOG_LEVEL?=fatal` so it applies to anything that make runs, then no need to repeat it in individual targets.